docs/erc1155-skills.md

```diff
@@ -0,0 +1,214 @@
+
+Generate and add a key for Ethereum use:
+``` bash
+aea generate-key ethereum
+aea add-key ethereum eth_private_key.txt
+```
+
+
+### Create the client AEA (ledger version)
+
+In another terminal, create the AEA that will sign the transaction.
+
+``` bash
+aea create my_erc1155_client
+cd my_erc1155_client
+aea add connection fetchai/oef:0.1.0
+aea add skill fetchai/erc1155_client:0.1.0
+aea add contract fetchai/erc1155:0.1.0
+aea install
+```
+
+Additionally, create the private key for the client AEA.
+
+Generate and add a key for Ethereum use:
+```bash
+aea generate-key ethereum
+aea add-key ethereum eth_private_key.txt
+```
+
+### Update the AEA configs
+
+Both in `my_erc1155_deploy/aea-config.yaml` and
+`my_erc1155_client/aea-config.yaml`, replace `ledger_apis: {}` with the following based on the network you want to connect
+
+Connect to Ethereum:
+``` yaml
+ledger_apis:
+  ethereum:
+    address: https://ropsten.infura.io/v3/f00f7b3ba0e848ddbdc8941c527447fe
+    chain_id: 3
+    gas_price: 50
+```
+
+### Update the deployer AEA skill configs
+
+In `my_erc1155_deploy/vendor/fetchai/skills/erc1155_deploy/skill.yaml`, update the details based on the following:
+``` yaml
+name: erc1155_deploy
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+fingerprint: {}
+description: "The erc1155 deploy skill implements the functionality to depoly and interact with a smart contract."
+contracts: ['fetchai/erc1155:0.1.0']
+behaviours:
+  service_registration:
+    class_name: ServiceRegistrationBehaviour
+    args:
+      services_interval: 60
+handlers:
+  default:
+    class_name: FIPAHandler
+    args: {}
+  transaction:
+    class_name: TransactionHandler
+    args: {}
+models:
+  strategy:
+    class_name: Strategy
+    args:
+      ledger_id: 'ethereum'
+      is_ledger_tx: True
+      nft: 1
+      ft: 2
+      item_ids: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+      mint_stock: [100, 100, 100, 100, 100, 100, 100, 100, 100, 100]
+      from_supply: 10
+      to_supply: 0
+      value: 0
+      search_schema:
+        attribute_one:
+          name: country
+          type: str
+          is_required: True
+        attribute_two:
+          name: city
+          type: str
+          is_required: True
+      search_data:
+        country: UK
+        city: Cambridge
+protocols: ['fetchai/fipa:0.1.0', 'fetchai/oef:0.1.0', 'fetchai/default:0.1.0']
+ledgers: ['fetchai']
+dependencies:
+  vyper: { version: "==0.1.0b12"}
+```
+The `search_schema` and the `search_data` are used to register the service in the OEF and make your agent discoverable. The name of each attribute must be a key in the `search_data` dictionary.
+
+
+### Fund the deployer AEA
+
+To create some wealth for your deployer AEA for the Ethereum `ropsten` network:
+
+``` bash
+aea generate-wealth ethereum
+```
+
+## Run the AEAs
+
+You can change the endpoint's address and port by modifying the connection's yaml file (my_seller_aea/connection/oef/connection.yaml)
```

Comment:
Let's just assume this is run from the same host so we can skip this.

Reply:
This is actually something that we add in every guide. That's why it is there :)